### ISSUE TITLE
Attribute for zone restorals

### ISSUE THREAD

**Issue by chelming** [ufodone/envisalink_new#112]
Is there any data (DSC) for the last time a zone was restored? The last tripped time is accurate and fantastic, but I don't have any good data for when the zone was restored. I can't go off of "updated" or "changed" because those will always be set when Home Assistant restarts.

![image](https://github.com/ufodone/envisalink_new/assets/7746625/065a9fa9-a048-453d-a071-494b2bf6354f)

**Comment by ufodone** [comment]
No, the envisalink does not provide that information.  The same actually applies to the last tripped time attribute too but the integration saves its state in HA so it is restored on restarts.  The caveat is that it's still possible to  have the wrong value if it was tripped and then restored while HA was down/restarting).

It might be possible to do something similar with the restore time but the same caveat would apply.

What do you mean by using "updated" or "changed"?

**Comment by chelming** [comment]
Updated and changed seem to be on all HA entities by default but they reset to the last boot time. E.g., this one shows that it changed yesterday but it was just a HA restart. 
![image](https://github.com/ufodone/envisalink_new/assets/7746625/ff5d7f8e-7f4d-4a8f-b540-d63e7757f75b)

**Comment by ufodone** [comment]
@chelming Given the caveat I mentioned above, would it still be useful to preserve this state across restarts?

**Comment by chelming** [comment]
I think so? honestly, I forgot what I was going to do with that attribute
